pycurious/grid.py

```diff
@@ -154,9 +150,9 @@ def create_centroid_list(self, window, spacingX=None, spacingY=None):
         yc = ycoords[n2w:-n2w]
 
         # but we can alter it if required
-        if type(spacingX) != type(None):
+        if spacingX is not None:
```

Comment:
It's better to check if the variable is `None` instead of comparing types. I've change this style on every occurrence.